src/components/RadioButton/RadioButton.st.css

```diff
@@ -198,8 +198,20 @@
 
 /* Create the indicator (the dot/circle - hidden when not checked) */
 
+
+.root:box:hover:not(:checked) .wrapper {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+    transition: border-color .2s ease-in-out;
+}
+
 .root:hover .radioIcon {
     border-color: applyOpacity(value(CurrentBorderColor), 1);
+    transition: border-color .2s ease-in-out;
+}
+
+.root:focus-within .radioIcon {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+    transition: border-color .2s ease-in-out;
 }
```

Comment:
Note that this will add the transition only on hover, but not when removing the cursor/focus from the element.
Is that ok?

Reply:
that's a good question. 
should I ask Johanna? what would you do different?

Reply:
Well, if we add it directly to the `radioIcon`, like this:
```css
.radioIcon {
  transition: border-color .2s ease-in-out;
}
```
It will affect both the enter and the leave.
However, this will also affect the border in all the radio-button's states: when the radio-button is being checked, disabled, etc.
so I guess it's best to leave it like this.
I would check with Johanna in any case, though I'm not sure achieving this is worth the effort.
(and I raised the issue in the first place!)